ui/shared/AccountSelector.qml

```diff
@@ -0,0 +1,201 @@
+                fillMode: Image.PreserveAspectFit
+                source: select.icon
+            }
+            ColorOverlay {
+                anchors.fill: iconImg
+                source: iconImg
+                color: iconColor
+            }
+            Column {
+                anchors.left: iconImg.right
+                anchors.leftMargin: 14
+                anchors.verticalCenter: parent.verticalCenter
+
+                StyledText {
+                    id: accountName
+                    text: name
+                    font.pixelSize: 15
+                    height: 22
+                }
+
+                StyledText {
+                    id: accountAddress
+                    text: address
+                    elide: Text.ElideMiddle
+                    width: 80
+                    color: Style.current.darkGrey
+                    font.pixelSize: 12
+                    height: 16
+                }
+            }
+            StyledText {
+                anchors.right: fiatCurrencySymbol.left
+                anchors.rightMargin: 4
+                anchors.verticalCenter: parent.verticalCenter
+                font.pixelSize: 15
+                height: 22
+                text: fiatBalance
+            }
+            StyledText {
+                id: fiatCurrencySymbol
+                anchors.right: parent.right
+                anchors.rightMargin: Style.current.padding
+                anchors.verticalCenter: parent.verticalCenter
+                font.pixelSize: 15
+                height: 22
+                color: Style.current.darkGrey
+                text: walletModel.defaultCurrency.toUpperCase()
+            }
+            background: Rectangle {
+                color: itemContainer.bgColor
+                radius: Style.current.radius
+
+                // cover bottom left/right corners with square corners
+                Rectangle {
+                    visible: !isLastItem
+                    anchors.left: parent.left
+                    anchors.right: parent.right
+                    anchors.bottom: parent.bottom
+                    height: parent.radius
+                    color: parent.color
+                }
+
+                // cover top left/right corners with square corners
+                Rectangle {
+                    visible: !isFirstItem
+                    anchors.left: parent.left
+                    anchors.right: parent.right
+                    anchors.top: parent.top
+                    height: parent.radius
+                    color: parent.color
+                }
+            }
+            MouseArea {
+                cursorShape: Qt.PointingHandCursor
+                anchors.fill: itemContainer
+                hoverEnabled: true
+                onEntered: {
+                    itemContainer.bgColor = Style.current.lightGrey
+                }
+                onExited: {
+                    itemContainer.bgColor = Style.current.white
```

Comment:
```suggestion
                    itemContainer.bgColor = Style.current.background
```

Reply:
Agreed, will handle this in updates to #698.

Reply:
👍